mmv1/third_party/terraform/resources/resource_storage_transfer_job.go

```diff
@@ -1153,3 +1204,45 @@ func flattenTransferJobNotificationConfig(notificationConfig *storagetransfer.No
 
 	return []map[string]interface{}{data}
 }
+
+func expandTransferJobLoggingConfig(loggingConfigs []interface{}) *storagetransfer.LoggingConfig {
+	if len(loggingConfigs) == 0 || loggingConfigs[0] == nil {
+		return nil
+	}
+
+	loggingConfig := loggingConfigs[0].(map[string]interface{})
+	logActionsInterface := loggingConfig["log_actions"].([]interface{})
+	logActions := make([]string, len(logActionsInterface))
+	for i, v := range logActionsInterface {
+		logActions[i] = fmt.Sprint(v)
+	}
+
+	LogActionStatesInterface := loggingConfig["log_action_states"].([]interface{})
+	LogActionStates := make([]string, len(LogActionStatesInterface))
+	for i, v := range LogActionStatesInterface {
+		LogActionStates[i] = fmt.Sprint(v)
+	}
+
+	var apiData = &storagetransfer.LoggingConfig{
+		LogActions:                  logActions,
+		LogActionStates:             LogActionStates,
+		EnableOnpremGcsTransferLogs: loggingConfig["enable_on_prem_gcs_transfer_logs"].(bool),
+	}
```

Comment:
This looks good, but for simplicity, we can just use `convertStringArr`, such as 
```suggestion
	var apiData = &storagetransfer.LoggingConfig{
		LogActions:                  convertStringArr(loggingConfig["log_actions"].([]interface{})),
		LogActionStates:             convertStringArr(loggingConfig["log_action_states"].([]interface{})),
		EnableOnpremGcsTransferLogs: loggingConfig["enable_on_prem_gcs_transfer_logs"].(bool),
	}
```